### PR TITLE
Separate MapRenderer utils.

### PR DESCRIPTION
**Pull request by siarheishut**

### CODE REVIEW

File: src/map_renderer_utils.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#include <vector>
+#include <string_view>
+#include <map>
+#include <unordered_map>
+
+#include "svg/common.h"
+
+#include "sphere.h"
+
+namespace rm::map_renderer_utils {
```
Comment (foxfrikses) [inline src/map_renderer_utils.h:13]: Maybe you can make the name shorter? Like `map_render`, or `render`, or `render_utils`?

File: src/map_renderer_utils.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+struct Route {
+  std::vector<std::string_view> route;
+  bool is_roundtrip;
+};
+using Buses = std::map<std::string_view, Route>;
+using Stops = std::map<std::string_view, rm::sphere::Coords>;
+using StopCoords = std::unordered_map<std::string_view, svg::Point>;
+}
+
+#endif // ROOT_MANAGER_SRC_MAP_RENDERER_UTILS_H_
```
Comment (foxfrikses) [inline src/map_renderer_utils.h:23]: endl

File: src/map_renderer_utils.h (hunk 1 above)
Comment (siarheishut) [reply src/map_renderer_utils.h:13]: I believe it's not a good idea to name it render, but I'd name it `renderer_utils`. WDYT?

Comment (siarheishut) [reply src/map_renderer_utils.h:13]: And anyway, I'm thinking about creating `renderer` namespace for class MapRenderer and just adding `using namespace renderer_utils;` in there to we could remove `renderer_utiils::` prefix, so a long name won't be a problem.

File: src/map_renderer_utils.h (hunk 2 above)
Comment (siarheishut) [reply src/map_renderer_utils.h:23]: Done.

File: src/map_renderer_utils.h (hunk 1 above)
Comment (foxfrikses) [reply src/map_renderer_utils.h:13]: Okay. You can also make it `renderer::utils`. Do whatever you like most. It's not a blocker of any kind for now.

**Review by foxfrikses — APPROVED** [review]
LGTM

File: src/map_renderer_utils.h (hunk 2 above)
Comment (siarheishut) [reply src/map_renderer_utils.h]: Done already.

**Pull request merged**